projects/igniteui-angular/src/lib/action-strip/grid-actions/grid-editing-actions.component.ts

```diff
@@ -128,14 +128,7 @@ export class IgxGridEditingActionsComponent extends IgxGridActionsBaseDirective
         this.strip.hide();
     }
 
-    /**
-     * Delete a row according to the context
-     *
-     * @example
-     * ```typescript
-     * this.gridEditingActions.deleteRowHandler();
-     * ```
-     */
+    /** @hidden @internal **/
     public deleteRowHandler(event?): void {
```

Comment:
@dkamburov Do check this one out - There's no usage of those 'handler' methods that I see in docs & samples, the readme doesn't describe them either. I left `startEdit` as other active had it I think and it makes some sense. I would accept public methods for these like `deleteRow` that just calls the internal handler